xapian-applications/omega/docs/newformat.rst

```diff
@@ -0,0 +1,156 @@
+=====================================
+Add support for a new format to Omega
+=====================================
+
+We can add support for a new file format to Omega through an external filter or a library. For this, we must follow a series of steps.
+
+First of all, we need a mime type for the new file format. Omega uses mime types to identify the format of a file and handle it in a proper way. The official registry for is at http://www.iana.org/assignments/media-types/ but not all filetypes have a corresponding official mime-type. In that case, a de-facto standard "x-" prefixed mime-type often exists. A good way to look for one is to ask the ``file`` utility to identify a file (Omega uses the same library as file to identify files when it does not recognise the extension)::
```

Comment:
Change:
* `official registry for is at` => `official registry is at`

Reply:
Okay, I am going to change it.